websites/T/Tracker.gg/presence.ts

```diff
@@ -0,0 +1,524 @@
+const presence = new Presence({ clientId: "929349462365704222" }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: browsingTimestamp
+    },
+    { pathname } = window.location,
+    botHost = document.location.hostname;
+
+  if (botHost === "tracker.gg") {
+    if (pathname === "/") {
+      presenceData.details = "Viewing Page:";
+      presenceData.state = "Homepage";
+    } else if (pathname === "/apps") {
+      presenceData.details = "Viewing Page:";
+      presenceData.state = "Tracker Apps";
+    } else if (pathname === "/developers") {
+      presenceData.details = "Viewing Page:";
+      presenceData.state = "Developers";
+    } else if (pathname === "/developers/apps/create") {
+      presenceData.details = "Creating";
+      presenceData.state = "Tracker Dev App";
+    } else if (pathname.includes("/developers/apps/")) {
+      try {
+        const appName = document
+          .querySelector<HTMLDivElement>(
+            "#app > div.trn-wrapper > div.trn-container > div > main > div.apps > div.content-container > div > div > div > div > div.dashboard__analytics.card.bordered.responsive > header > div > div.left > div:nth-child(2) > div.title"
+          )
+          .textContent.replace("Dashboard", "");
+        presenceData.details = "Editing App";
+        presenceData.state = appName;
+        presenceData.smallImageKey = "logo";
+        presenceData.smallImageText = appName;
+        presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+          "#app > div.trn-wrapper > div.trn-container > div > main > div.apps > div.content-container > div > div > div > div > div.dashboard__details.card.responsive > div > div:nth-child(1) > div > img"
+        ).src;
+      } catch (e) {
+        presenceData.details = "Viewing:";
+        presenceData.state = "Developer Apps";
+      }
+    } else if (pathname.includes("/developers/docs")) {
+      try {
+        const title: string = document.querySelector(
+          "#app > div.trn-wrapper > div.trn-container > div > main > div.docs > div.content-container > div > div > h1"
+        ).textContent;
+        presenceData.details = "Viewing Docs";
+        presenceData.state = title;
```

Comment:
Copy this throughout your code, L51

```suggestion
        presenceData.details = "Viewing Docs";
        presenceData.state = document.querySelector<HTMLHeadingElement>(
          "#app > div.trn-wrapper > div.trn-container > div > main > div.docs > div.content-container > div > div > h1"
        ).textContent;
```